packages/calcite-components/src/components/meter/meter.tsx

```diff
@@ -0,0 +1,291 @@
+import { Component, Element, Fragment, h, Host, Prop, State, VNode, Watch } from "@stencil/core";
+import { Appearance, Kind, Scale } from "../interfaces";
+import { CSS } from "./resources";
+
+import {
+  connectLocalized,
+  disconnectLocalized,
+  LocalizedComponent,
+  NumberingSystem,
+  numberStringFormatter,
+} from "../../utils/locale";
+
+@Component({
+  tag: "calcite-meter",
+  styleUrl: "meter.scss",
+  shadow: true,
+})
+export class Meter implements LocalizedComponent {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+  @Element() el: HTMLCalciteMeterElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  //--------------------------------------------------------------------------
+
+  /**
+   * When `true`, interaction is prevented and the component is displayed with lower opacity.
+   */
+  @Prop({ reflect: true }) disabled = false;
+
+  /**  Accessible name for the component. */
+  @Prop() label!: string;
+
+  /** Specifies the size of the component. */
+  @Prop({ reflect: true }) scale: Scale = "m";
+
+  /** Specifies the current value of the component. */
+  @Prop() value: number;
+
+  /** Specifies the miniumum possible value of the component. */
+  @Prop() min = 0;
+
+  /** Specifies the maximum possible value of the component. */
+  @Prop() max = 100;
+
+  /** Optionally provide a low value - above this, when `fillType` is 'range', the component display the appropriate color  */
+  @Prop() low: number;
+
+  /** Optionally provide a high value - above this, when `fillType` is 'range', the component display the appropriate color  */
+  @Prop() high: number;
+
+  /** Does the fill appear as a single "brand" color or does it display indications of low / high / "success", "danger", or "warning" based on low, high if provided, or value compared to max */
```

Comment:
WDYT about something like:

```Specifies the component's display, where `"single"` displays a single "brand" color and `"range"` displays a range of colors which can specify `low`, `high`, `min` or `max` values with "success", "warning", and "danger".```

Reply:
For brevity, what about this - I don't think listing brand / status colors are needed - those would be shown as tokens. 

`Specifies the component's display, where `"single"` displays a single color and `"range"` displays one of a range of colors when `low` or `high` are specified.

Reply:
Awesome! 🥇 

nitpick: on the display of props. Could backticks be added, in addition to the double quotes to all the values (e.g. `"single"`, `"range"`, etc.)?